src/Language/Haskell/Liquid/Constraint/Generate.hs

```diff
@@ -1551,6 +1551,7 @@ funExpr _ _
 
 untick :: CoreExpr -> CoreExpr
 untick (Tick _ e)  = untick e 
+untick (App e (Type _)) = untick e 
```

Comment:
Let me rename: I want to get 

compose @a @b @c f g :: {v:_ | v == compose f g}